### PR TITLE
[V2] Yocto BSP v5.9 kirkstone + Linux DDK v1.15 + other misc things

### PR DESCRIPTION
**Pull request by otyshchenko1**
This is V2 for https://github.com/xen-troops/meta-xt-rcar/pull/38 The WIP mask was lifted.

### CODE REVIEW

File: meta-xt-rcar-proprietary/recipes-graphics/gles-module/gles-module-egl-headers.bb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 PVRUM_URL ?= "git://git@gitpct.epam.com/epmd-aepr/pvr_um_vgpu_img.git"
-PVRUM_BRANCH = "1.11/5516664_5.1.0"
+PVRUM_BRANCH = "${@bb.utils.contains('XT_GUEST_INSTALL', 'doma', '1.11/5516664_5.1.0', '1.15/6052913_5.9.0', d)}"
```
Comment (lorc) [inline meta-xt-rcar-proprietary/recipes-graphics/gles-module/gles-module-egl-headers.bb:3]: I am wondering if it is correct thing to depend on DomA presence. What if we want to use DDK11 without DomA for some reason? How do you think, is it possible?

@rshym, what is your opinion?

I want to propose to add separate variable like `XT_USE_DDK11` or `XT_DDK_VERSION` to be able to chose DDK version independently of DomA presence.

File: meta-xt-rcar-proprietary/recipes-graphics/wayland/wayland-wsegl.bbappend [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+SRC_URI += "\
```
Comment (lorc) [inline meta-xt-rcar-proprietary/recipes-graphics/wayland/wayland-wsegl.bbappend:3]: In this cause you again making dependency on Android presence. Is this a good thing?

**Review by lorc — COMMENTED** [review]
Good job! 

I have only one question about the way we choose DDK version. In my opinion, it should be decoupled from Android presence for a better component separation. (Thanks to Xen PCI subsystem I am hating tightly coupled systems lately)

What is your opinion, guys?

File: meta-xt-rcar-proprietary/recipes-graphics/wayland/wayland-wsegl.bbappend (hunk 2 above)
Comment (otyshchenko1) [reply meta-xt-rcar-proprietary/recipes-graphics/wayland/wayland-wsegl.bbappend:3]: There is no dependency on DomA in current patch at all, at least an explicit dependency. The patch's purpose is just to apply an additional change on top of already separated from pvrum wayland-wsegl. The change is not DomA specific and should be applied to wayland-wsegl (for DDK v1.11 where the wayland-wsegl is a part of pvrum, the required change is applied there, at gitpct). The implicit dependency is in yaml which masks this recipe for DomA.

Comment (lorc) [reply meta-xt-rcar-proprietary/recipes-graphics/wayland/wayland-wsegl.bbappend:3]: Agree. But as I said in the other comments, we need to consider implementing more explicit way to chose DDK version.

Comment (otyshchenko1) [reply meta-xt-rcar-proprietary/recipes-graphics/wayland/wayland-wsegl.bbappend:3]: ok, I see. I will look into adding new XT_USE_DDK1_11 var

**Comment by lorc** [comment]
`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by otyshchenko1** [comment]
updated, only added A-b

**Review by lorc — APPROVED** [review]

**Pull request merged**